### PR TITLE
Collapse taskview

### PR DESCRIPTION
**Pull request by HishamAlmannai**

### CODE REVIEW

File: src/components/Task/Task.js [hunk 1]
```diff
@@ -1,48 +1,113 @@
...
 
 export default function Task(props) {
+	const {
+		task: { id, name, done, dueDate },
+	} = props;
+
 	const deleteTask = useStore(state => state.deleteTask);
 	const [editMode, setEditMode] = useState('');
+	const [activeMode, setActiveMode] = useState(true);
+	const [handleSubmit, setHandleSubmit] = useState(false);
```
Comment (mpagels) [inline src/components/Task/Task.js:17]: The naming `handleSubmit` is confusing. I would expect a `function` here and not a boolean. Highly recommend renaming this to more descriptive, that a colleague can easily understand, that this variable holds a `bool` value.

Variable that contains `boolean` should start with `is` or `has`
E.g.: `isActive`, `hasActiveMode`, `isInEditMode`, etc.

File: styles/Checkbox.styled.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const StyledCheckbox = styled.input`
+	height: 2rem;
+	border-radius: 5px;
+`;
+
+export default StyledCheckbox;
```
Comment (mpagels) [inline styles/Checkbox.styled.js:8]: Why do you name it `StyledCheckbox` when it is just a normal `input` field and not what the name indicates, a `checkbox`

**Review by mpagels — APPROVED** [review]
Good work here Hisham.

Have found some naming problems. Pls check.

File: styles/Checkbox.styled.js (hunk 2 above)
Comment (HishamAlmannai) [reply styles/Checkbox.styled.js]: This is a styling just for the checkbox. I have other styled inputs as well. I moved this to them.

File: src/components/Task/Task.js (hunk 1 above)
Comment (HishamAlmannai) [reply src/components/Task/Task.js]: Thanks for the remark. Changed it.

**Pull request merged**